msg/MyMessage.msg

```diff
@@ -0,0 +1,3 @@
+uint64 timestamp		# time since system start (microseconds)
+
+char[127] text
```

Comment:
Isn't this the same as https://github.com/PX4/PX4-Autopilot/blob/253208fdd4d3b415ef9ce019d7b5475a3e103a8a/msg/MavlinkLog.msg ?

Reply:
Actually no clue why this is here - will remove. Thanks for the catch!